galileo/examples/highlight_features.rs

```diff
@@ -0,0 +1,181 @@
+use std::sync::{Arc, RwLock};
```

Comment:
Please, add some summary information about the example to the `galileo/examples/README.md` file.